scripts/framework.lua

```diff
@@ -0,0 +1,19 @@
+Colors = {}
+
+function Set_colors()
```

Comment:
tu pourrais get plus de perf ici je crois

https://www.dannyguo.com/blog/how-to-concatenate-strings-in-lua/